README.md

```diff
@@ -17,6 +17,28 @@ Use the `-log.level` flag. Run the agent with the `-help` flag to see the possib
 | `info`       | 0 (`-v` not set) |
 | `debug`      | 3 (`-vvv`)       |
 
+## Environment Variables and Flags
+
+If using commandline flags is difficult to use in your environment, you can define environment variables to be consumed by the PDC Agent instead. *Commandline flags always take precedence.*
+
+The following mappings are currently provided:
+
+| Flag                        | Environment Variable                      |
+|-----------------------------|-------------------------------------------|
+| `-token`                    | `GCLOUD_PDC_SIGNING_TOKEN`                |
+| `-cluster`                  | `GCLOUD_PDC_CLUSTER`                      |
+| `-gcloud-hosted-grafana-id` | `GCLOUD_PDC_GCLOUD_HOSTED_GRAFANA_ID`     |
+| `-ssh-key-file`             | `GCLOUD_PDC_SSH_KEY_FILE`                 |
+| `-log-level`                | `GCLOUD_PDC_SSH_LOG_LEVEL`                |
+| `-skip-ssh-validation`      | `GCLOUD_PDC_SSH_SKIP_SSH_VALIDATION`      |
+| `-ssh-flag`                 | `GCLOUD_PDC_SSH_ADDITIONAL_SSH_FLAGS`†    |
+| `-force-key-file-overwrite` | `GCLOUD_PDC_SSH_FORCE_KEY_FILE_OVERWRITE` |
+| `-cert-expiry-window`       | `GCLOUD_PDC_SSH_CERT_EXPIRY_WINDOW`       |
+| `-cert-check-expiry-period` | `GCLOUD_PDC_SSH_CERT_CHECK_EXPIRY_PERIOD` |
+| `-metrics-addr`             | `GCLOUD_PDC_SSH_METRICS_ADDR`             |
```

Comment:
I should have caught this last time, but I don't think we need the `SSH` prefixing all of therse - it makes some of them stutter. Can we please have the format `GCLOUD_PDC_<camel case, to upper, of whatever the flag string is>?